internal/config/config.go

```diff
@@ -127,27 +127,40 @@ type SQLKotlin struct {
 }
 
 type Override struct {
-	// name of the golang type to use, e.g. `github.com/segmentio/ksuid.KSUID`
-	GoType string `json:"go_type" yaml:"go_type"`
+	// Import path, package name and type name as you would type them in the IDE
+	GoTypeParam GoTypeParams `json:"go_type" yaml:"go_type"`
```

Comment:
This is a backwards incompatible change. The `GoTypeParams` struct needs to work if `go_type` is a string. This will require a custom `UnmarshalJSON` method.